docs/api/synthetics/synthetics-api.asciidoc

```diff
@@ -0,0 +1,8 @@
+[[synthetics-apis]]
+== Synthetics APIs
+
+The following APIs are available for Synthetics.
+
+* <<add-monitor-api, Add a monitor>> - Add a new monitor to Synthetics app.
```

Comment:
Have you thought about how you would easily allow editing one of the monitors added via this api in an edit api?

I feel like that problem is solved in our project monitor schema by allowing people to edit things based off their specified custom monitor id (the name of the monitor or the id specified in the synthetics project). How can we make that easy here?